components/datatypes/consentpreferences.schema.json

```diff
@@ -0,0 +1,337 @@
+          "type": "object",
+          "properties": {
+            "xdm:dataCollection": {
+              "title": "Data Collection",
+              "description": "Data collection is permitted",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:sellingOrSharingOfData": {
+              "title": "Selling or Sharing of Data",
+              "description": "Sharing or Selling of Data with/to 2nd or 3rd parties is permitted, such as in compliance with CCPA",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:anonymousAnalysis": {
+              "title": "Anonymous Analysis",
+              "description": "Data can be used for analysis that does not require the use of any type of ID (whether it can be tied to an individual or not)",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:pseudonymousAnalysis": {
+              "title": "Pseudonymous Analysis",
+              "description": "Data can be used for analysis that may include an ID that may or may not be tied to an actual person",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:deviceLinking": {
+              "title": "Device Linking",
+              "description": "Attempts to associate multiple devices used/owned by an individual/household/organization are permitted",
+              "$ref": "#/definitions/consentOption"
+            }
+          }
+        },
+        "xdm:personalizationPreferences": {
+          "title": "Personalization Preferences",
+          "description": "User's Personalization Preferences",
+          "type": "object",
+          "properties": {
+            "xdm:general": {
+              "title": "General Personalization Preference",
+              "description": "Is there consent for personalization, unless overridden by a specific personalization preference",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:content": {
+              "title": "Personalize content",
+              "description": "Allow personalization of the content on my site or in my apps",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:inAppMessages": {
+              "title": "Personalize in-app messages",
+              "description": "Allow personalization of messages displayed within my apps",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:offers": {
+              "title": "Personalize Offers",
+              "description": "Allow personalization of offers (my own ads) displayed on my site or in my app",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:eMail": {
+              "title": "Personalize eMail",
+              "description": "Allow personalization of emails sent to the user",
+              "$ref": "#/definitions/consentOption"
+            },
+            "xdm:snailMail": {
```

Comment:
is this really an appropriate name?  it's a derogatory term, I wouldn't use it.

Reply:
What would you use instead? traditionalMail, physicalMail?

Reply:
physicalMail sounds good to me